compose.yml

```diff
@@ -51,41 +35,9 @@ services:
 
   liquibase:
     image: liquibase/liquibase
-    container_name: fcp-mpdp-backend-liquibase
     depends_on:
       postgres:
         condition: service_healthy
     volumes:
       - ./changelog:/liquibase/changelog
     command: --url=jdbc:postgresql://postgres:5432/fcp_mpdp_backend --username=postgres --password=postgres --changelog-file=changelog/db.changelog.xml update
-    networks:
-      - fcp-mpdp
-
-  localstack:
-    image: localstack/localstack:3.0.2
-    container_name: fcp-mpdp-backend-localstack
-    ports:
-      - '4567:4566' # LocalStack Gateway
-      - '4570-4619:4510-4559' # external services port range
-    env_file:
-      - 'compose/aws.env'
-    environment:
-      DEBUG: ${DEBUG:-1}
-      LS_LOG: WARN # Localstack DEBUG Level
-      SERVICES: s3,sqs,sns,firehose
-      LOCALSTACK_HOST: 127.0.0.1
-    volumes:
-      - '${TMPDIR:-/tmp}/localstack:/var/lib/localstack'
-      - ./compose/start-localstack.sh:/etc/localstack/init/ready.d/start-localstack.sh
-    healthcheck:
-      test: ['CMD', 'curl', 'localhost:4566']
-      interval: 5s
-      start_period: 5s
-      retries: 3
-    networks:
-      - fcp-mpdp
```

Comment:
Localstack removed as doesn't use any AWS components beyond Postgres which is already emulated.